src/main/java/ch/admin/bag/covidcertificate/client/inapp_delivery/internal/DefaultInAppDeliveryClient.java

```diff
@@ -89,7 +89,7 @@ private void logKpi(String code) {
             final var kpiTimestamp = LocalDateTime.now();
             log.info("kpi: {} {} {} {}", kv(KPI_TIMESTAMP_KEY, kpiTimestamp.format(LOG_FORMAT)),
                      kv(KPI_TYPE_KEY, KPI_TYPE_INAPP_DELIVERY), kv(KPI_UUID_KEY, extId), kv(KPI_DETAILS, code));
-            kpiLogService.saveKpiData(new KpiData(kpiTimestamp, KPI_TYPE_INAPP_DELIVERY, extId, null, code, null));
+            kpiLogService.saveKpiData(new KpiData(kpiTimestamp, KPI_TYPE_INAPP_DELIVERY, extId, null, code, null, false));
```

Comment:
Ist es tatsächlich nötig, dass wir "fraud" auch in den KPIs loggen? Aus meiner Sicht ist das redundant zum Eintrag in der Revocation-Tabelle.

Reply:
Ich bin da ähnlich unsicher wie Iris. Wir wollen mit der KPI für "re" sehen welche UVCI wann durch wen revoziert wurde. In der Revokations-Tabelle sehen wir dann den erstellten Eintrag mit der Information isFraud. Ob wir das in den KPI auch benötigen ist eben Ansichtssache. Jetzt ist es drin. Zurück bauen ist Aufwand aber auch nicht zu viel. Sollen wir es morgen kurz aus jassen :-)

Reply:
Können wir gerne tun.